.github/workflows/hacs_validation.yml

```
@@ -21,3 +22,18 @@ jobs:
         uses: "hacs/action@main"
         with:
           category: "integration"
+  
+  compatibility_check:
+    name: Quick Compatibility Check
+    runs-on: "ubuntu-latest"
+    steps:
+      - uses: "actions/checkout@v5"
+      - uses: actions/setup-python@v4
```

actionlint (1.7.8)

the runner of "actions/setup-python@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)